README.md

```diff
@@ -395,6 +397,9 @@ option() = {name, atom()} |
            {auto_ack, boolean()} |
            {ack_timeout, pos_integer()} |
            {force_ping, boolean()} |
+           {low_mem, boolean()} |
```

Comment:
this is for emqtt_bench, not necessarily used by other scenarios.

Reply:
OK, but we'd better list it here as we have this option.